### PR TITLE
Introduce new field `verified_at`.

### LINKED ISSUE #177: Switch from boolean to dateTime for verified field
Currently the verified field is a `boolean` type and does just say, that a user has been verified or not. Imagine to change the type to `dateTime` or add a new field `verified_at`. 

With this simple change we have the information _when_ the user has been verified. The check if the user is verified would be changed to `return is_null($this->verified_at)`.

### PR DESCRIPTION
**Pull request by Konafets**
The new field will be added next to the current field `verified`, which stores the information if a user has been verified its email address. Internally the new field is used for verification by check the field against `null`.

The new field makes it possible to not only store _if_ a user has been verified but also _when_.

Closes #177

### CODE REVIEW

File: src/UserVerification.php [hunk 1]
```diff
@@ -303,7 +306,7 @@ protected function getUserByEmail($email, $table)
      */
     protected function isVerified($user)
     {
-        if ($user->verified == true) {
+        if ($user->verified == true && !is_null($user->verified_at)) {
```
Comment (DariusIII) [inline src/UserVerification.php]: Maybe use $user->verified_at !== null?

**Comment by Konafets** [comment]
@jrean What is the status here? Do you consider to merge this?

**Comment by Konafets** [comment]
@jrean May I ask you why you are closing all these cool (mine and the others) PR without any explanation?

**Comment by jrean** [comment]
Hi @Konafets 
Because it is 2020 and this PR won't be merged as it it.
I wouldn't deal with this this way. I would rather change the code around the `verified` attribute to make it hold directly the timestamp or null.

If I update and implement it, I will not forget to credit your PR and your work.
Nothing wrong about closing a PR. It stays in the history of the project and I'm free to come back to it.

Thank you

**Pull request closed without merging**